When I got to my server/app/ folder I get the following error:
[pid: 15964|app: -1|req: -1/17] 192.168.1.98 () {42 vars in 686 bytes} [Sun Dec 30 18:51:55 2012] GET /app/ => generated 48 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 63 bytes (0 switches on core 0)
The strange thing is that this was working quite well until I decided to restart the uwsgi service.
[pid: 15963|app: -1|req: -1/18] 192.168.1.98 () {42 vars in 686 bytes} [Sun Dec 30 18:51:55 2012] GET /app/ => generated 48 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 63 bytes (0 switches on core 0)
[pid: 15963|app: -1|req: -1/16] 192.168.1.98 () {42 vars in 686 bytes} [Sun Dec 30 18:51:54 2012] GET /app/ => generated 48 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 63 bytes (0 switches on core 0)
[pid: 15963|app: -1|req: -1/15] 192.168.1.98 () {42 vars in 686 bytes} [Sun Dec 30 18:51:37 2012] GET /app/ => generated 48 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 63 bytes (0 switches on core 0)
Within my nginx sites-available/default this is the relevant portion for my app:
My app.xml file within the uwsgi/sites-available/ folder:
Finally my /web/NetWeaveCustom/index.py module is as follows:
There are no errors reported in the nginx error log.